internal/graph/check_test.go

```diff
@@ -996,6 +996,75 @@ condition condition1(param1: string) {
 	require.False(t, resp.Allowed)
 }
 
+func TestCheckDispatchCount(t *testing.T) {
+	ds := memory.New()
+	ctx := storage.ContextWithRelationshipTupleReader(context.Background(), ds)
+
+	storeID := ulid.Make().String()
+
+	model := parser.MustTransformDSLToProto(`model
+  schema 1.1
+
+type user
+
+type folder
+  	relations
+		define viewer: [user] or viewer from parent
```

Comment:
Please add a test that exercises `computed_userset` as well. For example:

```
type doc
  relations
     define owner: [user]
     define editor: [user] or owner
```

- document:1#owner@user:jon
- document:2#editor@user:will

`Check(document:1#owner@user:jon)` should require 1 dispatch.
`Check(document:2#editor@user:will)` should require 0 dispatches.

Evaluating a rewritten relation through computed_userset incurs a dispatch.